guides/hidden-pages.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: "How to add hidden pages"
+description: "Exclude pages from your docs.json"
+---
+
+Sometimes you may want to write a page and not publish it immediately.
+
+For content that is not strictly confidential, but isn't ready for prime time, you can add pages to your docs without referencing them in your `docs.json`. This is what is considered a "hidden page".
+
+As long as the page is not described under any "group", "tab" or other element in the navigation section of your `docs.json`, the only way a user will be able to navigate to the page is if they know or guess the page URL.
+
+<Note>
+Some navigation elements like sidebars, dropdowns, and tabs may shift or appear empty on hidden pages.
+</Note>
+
+If you want hidden pages to be indexed for SEO purposes and don't mind if their content appears in search results, you can add the following setting to your `docs.json`:
```

Comment:
```suggestion
If you want hidden pages to be indexed for SEO purposes and for their contents to appear when people search within your docs, add the following setting to your `docs.json`:
```

Combining some info in this section to make it a bit more concise. Had to split into two comments since the code block messes with GitHub's suggestion formatting.